app/models/solr_document.rb

```diff
@@ -46,6 +26,24 @@ class SolrDocument
   # Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
 
+  # Blacklight OAI metadata mapping
+  field_semantics.merge!(
+    title:       "title_tesim",
+    description: "description_tesim",
```

Comment:
Typically, all notes fields (including description) would map to dc:description. Is this just our description field?

Reply:
The mapping can use multiple fields so I just need to know which ones to use

Reply:
The fields would be:
- description
- note + all of the "typed" note fields (I think these are all included currently in "note_label_tesim")
- preferred citation